server/src/test/java/org/candlepin/dto/api/v1/NestedOwnerDTOTranslatorTest.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2009 - 2020 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.dto.api.v1;
+
+import org.candlepin.dto.AbstractTranslatorTest;
+import org.candlepin.dto.ModelTranslator;
+import org.candlepin.model.Owner;
+
+import static org.junit.Assert.*;
+
+/**
+ * Test suite for the NestedOwnerDTOTranslator class.
+ */
+
```

Comment:
There shouldn't be a space between class javadoc and class definition. What I meant in the other comments was that there should be space between class definition and the next thing (in that case the `@Override` annotation of the first method

Reply:
Understood, done.